tests/test_datatransport_meas_report/README.md

```diff
@@ -0,0 +1,58 @@
+<!-- 
+To compile puml use: (Assuming plantuml you are in the directory)
+plantuml.jar -tsvg README.md -o diagrams
+-->
+
+
+# Test on the DataTransport package
+
+## Todo
+
+## What measurements are interesting
+
+| Measurement | Description |
+|---|---|
+| Delay | Test the round-trip/2, in order to know it |
+| Jitter | When testing round-trip time log the jitter |
+
+## Test that will deliver the measurements
+
+One test should be able to deliver both measurements.
+
```

Comment:
Der skal ikke stå måske, der skal stå hvad der sket. Så hvis ved at køre test_datatransport_slave at målinger for begge test fåes, skal der bare stå det